faqs.md

```diff
@@ -58,7 +65,7 @@ We’re in a privileged position to push forward and raise awareness about great
 
 ### **Community**
 
-> Once we have the [Aragon Voting App](../dev/apps/voting.md) and a feedback mechanism from the community, all of ANT holders will get to participate in the decision making!
+We are working on transitioning Aragon’s governance from a highly centralized model to a more decentralized one and we will be experiment with community participation in the Nest program. 
```

Comment:
Should be either "be experimenting" or "will experiment"

Reply:
Oh sure that's a typo